vuu-ui/packages/vuu-popups/src/notifications/NotificationsProvider.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState, useContext, useCallback, useEffect } from "react";
+import classNames from "classnames"
+import { getUniqueId } from "@finos/vuu-utils";
+
+import "./notifications.css"
+
+// animation times in milliseconds
+const toastDisplayDuration = 6000;
```

Comment:
I think the gif in the PR description was made before this was changed. Could you recreate it so we can see how it looks at 6 seconds?